lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -262,34 +265,59 @@ class CategoryRenderer {
       case 'accessibility':
         return this._renderAccessibilityCategory(category, groups);
       default:
-        return this._renderDefaultCategory(category);
+        return this._renderDefaultCategory(category, groups);
     }
   }
 
   /**
    * @param {!ReportRenderer.CategoryJSON} category
+   * @param {!Object<string, !ReportRenderer.GroupJSON>} groupDefinitions
    * @return {!Element}
    */
-  _renderDefaultCategory(category) {
+  _renderDefaultCategory(category, groupDefinitions) {
     const element = this._dom.createElement('div', 'lh-category');
     element.id = category.id;
     element.appendChild(this._renderCategoryScore(category));
 
-    const passedAudits = category.audits.filter(audit => audit.score === 100);
-    const nonPassedAudits = category.audits.filter(audit => !passedAudits.includes(audit));
+    const manualAudits = category.audits.filter(audit => audit.result.manual);
+    const nonManualAudits = category.audits.filter(audit => !manualAudits.includes(audit));
+    const passedAudits = nonManualAudits.filter(audit => audit.score === 100);
+    const nonPassedAudits = nonManualAudits.filter(audit => !passedAudits.includes(audit));
 
     for (const audit of nonPassedAudits) {
       element.appendChild(this._renderAudit(audit));
     }
 
-    // Don't create a passed section if there are no passed.
-    if (!passedAudits.length) {
-      return element;
+    // Create a passed section if there are passing audits.
+    if (passedAudits.length) {
+      const passedElements = passedAudits.map(audit => this._renderAudit(audit));
+      const passedElem = this._renderPassedAuditsSection(passedElements);
+      element.appendChild(passedElem);
     }
 
-    const passedElements = passedAudits.map(audit => this._renderAudit(audit));
-    const passedElem = this._renderPassedAuditsSection(passedElements);
-    element.appendChild(passedElem);
+    // Render manual audits after passing.
+    const auditsGroupedByGroup = /** @type {!Object<string,
+        !Array<!ReportRenderer.AuditJSON>>} */ ({});
+    manualAudits.forEach(audit => {
```

Comment:
this feels a bit like overkill right now, do we think we have any need for manual audits to be broken up in the same category?